priiloader/hacks_hash.ini

```diff
@@ -548,3 +548,15 @@ minversion=1
 amount=1
 hash=0x2c030000,0x418200ac,0x38000001,0x981f0010
 patch=0x480000b0
+[No-Block RAAE,408x,410x]
+maxversion=513
+minversion=417
+amount=1
+hash=0x28009ea3,0x4082009c
+patch=0x28009ea3,0x4082009c,0x4280006c
```

Comment:
i also had another look at this. despite the function calls being strncmp i would like you to change (and test) the patch with the following :

amount=1
hash=0x2c000000,0x41820024,0x387e0717
patch=0x2c000000,0x60000000